pkg/jwks/jwks.go

```diff
@@ -136,7 +136,7 @@ func parseKeys(data []byte) ([]string, map[string]string, error) {
 		if err != nil {
 			return nil, nil, fmt.Errorf("marshal error: %w", err)
 		}
-		keys[i] = string(pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: encoded}))
+		keys[i] = strings.TrimSpace(string(pem.EncodeToMemory(&pem.Block{Type: "PUBLIC KEY", Bytes: encoded})))
```

Comment:
Should we copy over the project-level helper from the verification server instead? I'm fine with a copy-paste implementation.

Reply:
added... throughout the codebase